tools/ubergen/bin/ubergen.ts

```diff
@@ -78,23 +84,29 @@ function findWorkspacePath(): string {
   }
 }
 
-async function findLibrariesToPackage(): Promise<readonly LibraryReference[]> {
+async function findLibrariesToPackage(uberPackageJson: PackageJson): Promise<readonly LibraryReference[]> {
   console.log('🔍 Discovering libraries that need packaging...');
 
+  const deprecatedPackages = uberPackageJson.ubergen?.deprecatedPackages;
   const result = new Array<LibraryReference>();
   const librariesRoot = path.resolve(ROOT_PATH, 'packages', '@aws-cdk');
 
   for (const dir of await fs.readdir(librariesRoot)) {
     const packageJson = await fs.readJson(path.resolve(librariesRoot, dir, 'package.json'));
 
     if (packageJson.ubergen?.exclude) {
-      console.log(`\t⚠️ Skipping (ubergen excluded): ${packageJson.name}`);
-      continue;
-    } else if (packageJson.deprecated) {
-      console.log(`\t⚠️ Skipping (deprecated):       ${packageJson.name}`);
+      console.log(`\t⚠️ Skipping (ubergen excluded):   ${packageJson.name}`);
       continue;
     } else if (packageJson.jsii == null ) {
-      console.log(`\t⚠️ Skipping (not jsii-enabled): ${packageJson.name}`);
+      console.log(`\t⚠️ Skipping (not jsii-enabled):   ${packageJson.name}`);
+      continue;
+    } else if (deprecatedPackages) {
```

Comment:
```suggestion
    } else if (deprecatedPackages !== undefined) {
```

so that empty array signifies, don't deprecate anything.

Reply:
That's actually the behavior today, as [an empty array is truthy in TypeScript](https://www.typescriptlang.org/play?#code/MYewdgzgLgBAbjAvDAFAbQLoEoYH4YBEApgLYAOUAnjAIYBOdN1AlhDFHQK5QAWlBMAFyFSFavUYs2AMxoAbCPwDcAKFCQQcogDo5IAOYo4WFUA).